src/TimeModeling/judiWeights.jl

```diff
@@ -5,6 +5,8 @@
 # Authors: Philipp Witte (pwitte@eos.ubc.ca), Henryk Modzelewski (hmodzelewski@eos.ubc.ca)
 # Date: June 2019
 
+import Base.maximum
+import Base.minimum
```

Comment:
imports go in `JUDI.jl` not here. Thought was working out of the box with indexing defined and would be necessary,